frontend/src/components/samples/SampleTableColumns.tsx

```diff
@@ -49,12 +49,17 @@ export enum SampleColumnID {
 	SAMPLE_COUNT = 'SAMPLE_COUNT',
 }
 
+const SMALL_COLUMN_WIDTH = 90
```

Comment:
Maybe we need TINY around 40 ? This system is not very flexible but it simplify the management ... We could add a couple more steps to make it easier to find one that fits.